implementations/rust/ockam/ockam_command/src/node/start.rs

```diff
@@ -67,6 +67,7 @@ async fn run_impl(
         None,
         None, // No launch config available
         None,
+        None,
```

Comment:
Any chance we could annotate more of the Nones with a quick comment, at least the newly introduced one?